testutils/clustermanager/e2e-tests/config.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clustermanager
+
+import (
+	"regexp"
+
+	"knative.dev/pkg/testutils/clustermanager/e2e-tests/boskos"
+)
+
+const (
+	DefaultGKEMinNodes  = 1
+	DefaultGKEMaxNodes  = 3
+	DefaultGKENodeType  = "e2-standard-4"
+	DefaultGKERegion    = "us-central1"
+	DefaultGKEZone      = ""
```

Comment:
why this is named?

Reply:
This is actually copied from another file. By default the tool will create regional cluster, but we can also specify the zone.

Reply:
just do `"" /*zone*/` in the callsite.